### ISSUE TITLE
Sphinx causes an infinite loop in trace_requirements

### ISSUE THREAD

**Issue by NorthIsUp** [Yelp/pip-faster#52]
requirements.txt
```
sphinx
```
`$ python venv_update.py test_venv_run requirements.txt`

will hang in the while loop in `trace_requirements` as it seems sphinx has a requirement loop.

specifically because sphinx depends on sphinx-rtd-them and vice versa.

```
sphinx-rtd-theme>=0.1,<0.2 (from sphinx (from -r requirements.txt (line 1)))
sphinx>=1.1 (from sphinx-rtd-theme>=0.1,<0.2 (from sphinx (from -r requirements.txt (line 1))))
```

**Comment by struys** [comment]
We ran into this just now.

Let's bump the priority up a bit.

**Comment by NorthIsUp** [comment]
Booyah!

-adam🚀




On Wed, Nov 4, 2015 at 5:18 PM -0800, "Ken Struys" <notifications@github.com> wrote:












We ran into this just now.



Let's bump the priority up a bit.

**Comment by chriskuehl** [comment]
this is still a bug -.-

**Comment by chriskuehl** [comment]
fixed in #93
